src/libs/fsdrv/lv_fs_littlefs.c

```diff
@@ -0,0 +1,165 @@
+#include "../../../lvgl.h"
+#if LV_USE_FS_LITTLEFS != '\0'
+
+#include <LittleFS.h>
+#include "../../core/lv_global.h"
+
+struct LittleFile {
+    File file;
+};
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+static void fs_init(void);
+static void * fs_open(lv_fs_drv_t * drv, const char * path, lv_fs_mode_t mode);
+static lv_fs_res_t fs_close(lv_fs_drv_t * drv, void * file_p);
+static lv_fs_res_t fs_read(lv_fs_drv_t * drv, void * file_p, void * buf, uint32_t btr, uint32_t * br);
+static lv_fs_res_t fs_write(lv_fs_drv_t * drv, void * file_p, const void * buf, uint32_t btw, uint32_t * bw);
+static lv_fs_res_t fs_seek(lv_fs_drv_t * drv, void * file_p, uint32_t pos, lv_fs_whence_t whence);
+static lv_fs_res_t fs_tell(lv_fs_drv_t * drv, void * file_p, uint32_t * pos_p);
+
+/**
+ * Register a driver for the LittleFS File System interface
+ */
+void lv_port_littlefs_init(void)
+{
+    fs_init();
+
+    /*Add a simple drive to open images*/
+    static lv_fs_drv_t fs_drv;
+    lv_fs_drv_init(&fs_drv);
+
+    fs_drv.letter = LV_FS_LITTLEFS_LETTER;
+    fs_drv.open_cb = fs_open;
+    fs_drv.close_cb = fs_close;
+    fs_drv.read_cb = fs_read;
+    fs_drv.write_cb = fs_write;
+    fs_drv.seek_cb = fs_seek;
+    fs_drv.tell_cb = NULL;
```

Comment:
Please implement `tell_cb` also, as many internal code use it, for example the image decoders:
- https://github.com/lvgl/lvgl/blob/master/src/libs/gif/gifdec.c#L779
- https://github.com/lvgl/lvgl/blob/master/src/libs/libpng/lv_libpng.c#L205
- https://github.com/lvgl/lvgl/blob/master/src/libs/lodepng/lodepng.c#L374
- https://github.com/lvgl/lvgl/blob/master/src/libs/libjpeg_turbo/lv_libjpeg_turbo.c#L218